modules/monitoring-attaching-additional-labels-to-your-time-series-and-alerts.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc
+
+[id="attaching-additional-labels-to-your-time-series-and-alerts_{context}"]
+= Attaching additional labels to your time series and alerts
+
+Using the external labels feature of Prometheus, you can attach additional custom labels to all time series and alerts leaving the Prometheus cluster.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
+
+.Procedure
+
+. Start editing the `cluster-monitoring-config` ConfigMap:
```

Comment:
Suggest just "Edit the"

Reply:
I'll rather keep this, so that it means specifically "commence" as opposed to the more general "edit". It fits in with the last step "Save the file to apply the changes..." which I forgot :) This is also consistent with the way the rest of the Monitoring docs are.